metrics/domain.go

```diff
@@ -11,32 +11,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package domain
+package metrics
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
-	loadSchemaCounter = prometheus.NewCounterVec(
+	// LoadSchemaCounter records the counter of load schema.
+	LoadSchemaCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: "tidb",
 			Subsystem: "domain",
 			Name:      "load_schema_total",
 			Help:      "Counter of load schema",
 		}, []string{"type"})
 
-	loadSchemaDuration = prometheus.NewHistogram(
+	// LoadSchemaDuration records the duration of load schema.
+	LoadSchemaDuration = prometheus.NewHistogram(
 		prometheus.HistogramOpts{
 			Namespace: "tidb",
 			Subsystem: "domain",
 			Name:      "load_schema_duration",
 			Help:      "Bucketed histogram of processing time (s) in load schema.",
 			Buckets:   prometheus.ExponentialBuckets(0.001, 2, 15),
 		})
+
+	// LoadPrivilegeCounter records the counter of load privilege.
+	LoadPrivilegeCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "domain",
+			Name:      "load_privilege_total",
```

Comment:
s/load_privilege_total/load_privilege ?

Reply:
It's a counter, so `_total` is OK, you can see similar name rules in their best practice:
https://prometheus.io/docs/practices/naming/